docs/cep/table/index.md

```diff
@@ -51,7 +51,7 @@ FROM from_stream … ;
 You can also use general store syntax:
 
 ```sql
-CREATE STORE (GLOBAL|LOCAL)? <table_name> WITH(type="database", propKey=”propVal”, … , PrimaryKey='<attribute_name>', Index='<attribute_name>')(<attribute_name> <attribute_type>, ...);
+CREATE STORE (GLOBAL|LOCAL)? <table_name> WITH(type="database", propKey=”propVal”, … , Index='<attribute_name>')(<attribute_name> <attribute_type>, ...);
```

Comment:
@diana-macrometa - We cannot create inline indexes for `c8db` and you should also take out `Index='<attribute_name>'...`.

Reply:
What do you mean "inline indexes"?

Updated this line, btw.

Reply:
@diana-macrometa - The RDBMS allow the indexes to be created along with the table definition. Such indexes are named `inline`. However this is not valid for  `c8db`.